docs/stencil-docs/reference-docs/handlebars-helpers-reference.md

```diff
@@ -1226,6 +1227,39 @@ Strips query string from a URL.
 
 - [See it in GitHub](https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/stripQuerystring.js)
 
+### {{strReplace}}
+
+```handlebars
+{{strReplace string substr newSubstr occurrence}}
+```
+
+Replaces some or all occurrences of substring within the given string.
+
+#### Parameters
+
+- `string` {String}: The original string.
+- `substr` {String}: The substring in `string` to be replaced.
+- `newSubstr` {String}: The substring to replace `substr`.
+- `occurence` {Number}: The number of occurrences of `substr` to be replaced by `newSubstr` in `string`.

```

Comment:
```suggestion
- `occurrenceSelection` {Integer}: Optional. The number of occurrences of `targetSubstring` to replace with `newSubstring`. If you pass no argument or the supplied integer is greater than the number of `targetSubstring` occurrences, `newSubstring` will replace all instances of `targetSubstring`. A positive integer will replace `targetSubstring` starting from the beginning of the line;  a negative integer will replace from the end of the line.

```

Reply:
test end of line condition and modify this description or line 1251-1252 accordinly.

Reply:
suggestion updated following collaboration.